client/components/AllProducts-orig.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
```

Comment:
Same with this. With git, there's no reason to ever keep around something like an "original" version of a file. Because git is _version_ control, so you can always jump back in time to that "original version." Even when your main branch moves forward, no work committed to a git repository is ever lost because that's what version control does, it gives us a way to navigate through different versions of our app at different points in time